dokku

```diff
@@ -137,7 +137,13 @@ case "$1" in
 
         if [[ -z "$DOKKU_HEROKUISH" ]]; then
           DOKKU_DOCKERFILE_PORT=$(dokku config:get $APP DOKKU_DOCKERFILE_PORT || true)
-          START_CMD=$(dokku config:get $APP DOKKU_DOCKERFILE_START_CMD || $START_CMD)
+          if [[ -z "$START_CMD" ]]; then
+            START_CMD=$(dokku config:get $APP DOKKU_DOCKERFILE_START_CMD)
+          fi
+          if [[ -z "$START_CMD" ]]; then
+            # get cmd from Procfile
+            START_CMD=$(get_cmd_from_procfile $APP $PROC_TYPE)
+          fi
```

Comment:
You'll also need to add this line under the source command for `common/functions`

```
source "$PLUGIN_ENABLED_PATH/common/functions"
```